src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -338,30 +343,30 @@ listed below.
+| y                | REQUIRED              | Y position.                                                                                                                      |
+| z                | REQUIRED              | Z position. If electrodes are in 2D space this should be a column of `n/a` values.                                               |
+| size             | REQUIRED              | Surface area of the electrode, units MUST be in `mm^2`.                                                                          |
 
 SHOULD be present:
 
-| Column name  | Definition                                                                                                                                                           |
-| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| material     | OPTIONAL. Material of the electrodes.                                                                                                                                |
-| manufacturer | OPTIONAL. Recommended field to specify the manufacturer for each electrode. Can be used if electrodes were manufactured by more than one company.                    |
-| group        | OPTIONAL. Optional field to specify the group that the electrode is a part of. Note that any group specified here should match a group specified in `_channels.tsv`. |
-| hemisphere   | OPTIONAL. Optional field to specify the hemisphere in which the electrode is placed, one of `['L' or 'R']` (use capital).                                            |
+| **Column name**  | **Requirement level** | **Description**                                                                                                                                               |
+| ---------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| material         | RECOMMENDED           | Material of the electrodes.                                                                                                                                   |
+| manufacturer     | RECOMMENDED           | Recommended field to specify the manufacturer for each electrode. Can be used if electrodes were manufactured by more than one company.                       |
+| group            | RECOMMENDED           | RECOMMENDED field to specify the group that the electrode is a part of. Note that any group specified here should match a group specified in `_channels.tsv`. |
+| hemisphere       | RECOMMENDED           | RECOMMENDED field to specify the hemisphere in which the electrode is placed, one of `['L' or 'R']` (use capital).                                            |
```

Comment:
Tried to remove the right number of spaces/dashes, but I might have gotten it wrong.

```suggestion
| **Column name**  | **Requirement level** | **Description**                                                                                                                  |
| ---------------- | --------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
| material         | RECOMMENDED           | Material of the electrodes.                                                                                                      |
| manufacturer     | RECOMMENDED           | The manufacturer for each electrode. Can be used if electrodes were manufactured by more than one company.                       |
| group            | RECOMMENDED           | The group that the electrode is a part of. Note that any group specified here should match a group specified in `_channels.tsv`. |
| hemisphere       | RECOMMENDED           | The hemisphere in which the electrode is placed, one of `['L' or 'R']` (use capital).                                            |
```